scripts/g.extension/g.extension.py

```diff
@@ -1273,6 +1273,63 @@ def install_extension_xml(edict):
     return None
 
 
+def filter_multi_addon_addons(mlist):
+    """Filter out list of multi-addon addons which contains
+    and installs only *.html manual page, without source/binary
+    excutable module and doesn't need to check metadata.
+
+    e.g. i.sentinel multi-addon install i.sentinel addon which
+    install only i.sentinel.html file.
```

Comment:
```suggestion
    e.g. the i.sentinel multi-addon consists of several full i.sentinel.* addons
    along with a i.sentinel.html overview file.
```

... as a suggestion.